pkg/naming/naming.go

```diff
@@ -12,3 +18,26 @@ const (
 const (
 	ContainerBackupAgent = "backup-agent"
 )
+
+const (
+	ComponentMongod    = "mongod"
+	ComponentMongos    = "mongos"
+	ComponentNonVoting = "nonVoting"
+	ComponentArbiter   = "arbiter"
+)
+
+func MongodStatefulSetName(cr *psmdbv1.PerconaServerMongoDB, rs *psmdbv1.ReplsetSpec) string {
+	return fmt.Sprintf("%s-%s", cr.Name, rs.Name)
+}
+
+func MongosStatefulSetName(cr *psmdbv1.PerconaServerMongoDB) string {
```

Comment:
this is never used.

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1861/commits/b5ff21ce2bedc65d6ce7807638eb52f7ce1ef00b